checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -32,6 +40,7 @@
  *         <li value="2">array creation tree of size 0, e.g. {@code c.toArray(new String[0])}, or
  *         <li value="3">array creation tree of the collection size method invocation {@code
  *             c.toArray(new String[c.size()])}
+ *         <li value="4">field access where the declaration has a @ArrayLen(0) annotation
```

Comment:
Use `{@code ..}`

Reply:
Don't worry about it, I will clean it soon.